README.md

```diff
@@ -1,13 +1,19 @@
 # PayPal JS
 
-An async loader for the [PayPal JS SDK](https://developer.paypal.com/docs/checkout/).
+A client-side loader for the [PayPal JS SDK](https://developer.paypal.com/docs/checkout/).
```

Comment:
The thinking here is we should define paypal-js as a client-side loader that's used in the browser. In the future we may have a separate project for a server-side loader and we'd like to avoid confusion over these two different use cases.

Reply:
@gregjopa Do we want to link to the new JS SDK docs or to just checkout?

Reply:
@mnicpt that's a great idea. I'll send a new PR for updating the links before cutting a new version.